InternetHospital.BusinessLogic/Services/PatientService.cs

```diff
@@ -24,6 +25,59 @@ public PatientService(ApplicationContext context, IFilesService uploadingFiles)
             _uploadingFiles = uploadingFiles;
         }
 
+        public async Task<PatientDetailedModel> Get(int id)
+        {
+            PatientDetailedModel returnedPatient = null;
+            var searchedPatient = await _context.Users.Include(p=> p.IllnessHistories).FirstOrDefaultAsync(pa=> pa.Id == id);
+            if (searchedPatient != null)
+            {  
+                returnedPatient = Mapper.Map<User, PatientDetailedModel>(searchedPatient);
+            }
+            return returnedPatient;
+        }
+        public (IEnumerable<IllnessHistoryModel> histories, int count) GetFilteredHistories(IllnessHistorySearchModel queryParameters)
+        {
+            var histories = _context.IllnessHistories.AsQueryable();
```

Comment:
Unnecessary AsQueryable() because LINQ to Entity return IQueryable by default

Reply:
it returns ISortedQueryable, but i need IQueryable to use Where()